front/app/components/EventCards/messages.ts

```diff
@@ -33,13 +33,13 @@ export default defineMessages({
     id: 'app.components.EventCard.a11y_readMore',
     defaultMessage: 'Read more about the "{eventTitle}" event.',
   },
-  attending: {
-    id: 'app.containers.EventsShow.attending',
-    defaultMessage: 'attending',
+  haveRegistered: {
+    id: 'app.containers.EventCard.haveRegistered',
+    defaultMessage: 'have registered',
   },
-  attended: {
-    id: 'app.containers.EventsShow.attended',
-    defaultMessage: 'attended',
+  registered: {
+    id: 'app.containers.EventCard.registered',
+    defaultMessage: 'registered',
```

Comment:
Not sure I like these. I'm trying to accommodate Nola's point that using 'attended' causes confusion (as people have not necessarily attended anything).

My concern is this works in English, but the tense translations may not be at all obvious without context.